site/content/blog/2021-02-01-whats-new-in-svelte-february-2021.md

```diff
@@ -0,0 +1,74 @@
+---
+title: What's new in Svelte: February 2021
+description: A Svelte-packed showcase to kick-off the new year!
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+With the shortest month of the year coming up, Svelte maintainers and community members alike have been busy this last month – from new aria roles in the compiler to semantic highlighting hooks in the langugage-tools. Meanwhile, lots of folks have been busy integrating Svelte with other popular frameworks.
+
+## New compiler features
+- Aria roles from the [WAI-ARIA Graphics Module](https://www.w3.org/TR/graphics-aria-1.0/#role_definitions) are now recognized as valid aria roles in Svelte components (**3.31.1**)
+- Compiler warnings for the common React attributes, `className` and `htmlFor`, now make it easier to port React components to Svelte (**3.31.1**)
+
+Have a suggestion for a compiler feature or want to help implement new features/bug fixes? Check out the ["triage: good first issue" tag for Svelte](https://github.com/sveltejs/svelte/issues?q=is%3Aopen+is%3Aissue+label%3A%22triage%3A+good+first+issue%22)
+
+
+## New bits in language-tools
+
+- User disabled auto import suggestions no longer show in VS Code (**103.0.0**)
+- Renaming a variable is now safer with smart additions of a prefix/suffix to renamed variables (**104.0.0**)
+- Semantic (token) highlighting for typescript users lets theme makers apply Svelte-specific styling in their themes (**104.0.0**)
+- "Extract Component" has been added to the context menu - allowing you to extract components out of files without having to open the command window to type "Svelte: Extract Component" (**104.0.0**)
+
+For the complete list of changes, check out the language-tools [Releases page](https://github.com/sveltejs/language-tools/releases).
+
+A great way to try the language tools is to download the [Svelte Extension for VSCode](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode). This extension provides syntax highlighting and rich intellisense for Svelte components in VS Code, using the svelte language server. Check your editor's extension sources to see if there's a Svelte plugin for your IDE or build your own (see [coc-svelte](https://github.com/coc-extensions/coc-svelte) for example)!
+
```

Comment:
```suggestion

## New major version of `prettier-plugin-svelte`

`prettier-plugin-svelte` version 2 was released. It received an overhaul and comes with a rewrite of the HTML formatting. The output is now much more in line with how standard Prettier formats HTML. This is also why `svelteBracketNewLine` now defaults to `true`. Another notable default change is the sort order: `svelte:options` is now part of the sort order and the default changed to `options-scripts-markup-styles`, which is in line with how the majority of users like to order the code blocks. Additionally, Prettier's `htmlWhitespaceSensitivity` setting is now supported.

```

Reply:
Thanks! Definitely open to suggestions on what to highlight in svelte-loader, sapper, rollup-plugin-svelte as well ( @benmccann )
I can write a quick summary later this week based on the changelogs :)